src/containers/DeveloperPortalProjects/components/Breadcrumb/index.tsx

```diff
@@ -146,7 +146,7 @@ const Breadcrumb: FC<Props> = ({breadcrumbHeight, className}) => {
                   transformOrigin={{horizontal: 'center', vertical: 'top'}}
                   transformOffset={{horizontal: 50, vertical: 12}}
                 >
-                  {PATHNAME_TO_DROPDOWN_SELECTIONS[pathname].map((selection) => {
+                  {PATHNAME_TO_DROPDOWN_SELECTIONS[pathname]?.map((selection) => {
```

Comment:
@jamessspanggg this will prevent pages from crashing on small screens(until we fill up the constants file).

Reply:
yep, can we merge this https://github.com/thenewboston-developers/Website/pull/2011 first? It adds the project list in the breadcrumb and sidemenu